templates/extras/calc.md

```diff
@@ -23,26 +23,30 @@ end
 ```
 In fact, `calc A = B : H ...` in tactic mode functions exactly like a call to `refine (calc A = B : H ...)`.
 
-## Error messages, and how to avoid them
+## Getting effective feedback while using calc
 
-Note that the error messages can be quite obscure when things aren't quite right, and often the red
-squiggles end up under a random `...`. A tip to avoid these problems with calc usage is to first
+To get helpful error messages, keep the calc structure even before the proof is complete. Use `_` as in the 
+example above or `sorry` to stand for missing justifications. `sorry` will supress error messages entirely, 
+while `_` will either generate a nice error message, or in tactic mode, a new subgoal. 
+
+If the structure of calc is incorrect (e.g., (missing `:` or the justification after it), you may see error 
+messages that are obscure, and/or red squiggles end up under a random `...`. To avoid these, you might first 
```

Comment:
For parallelism with "you may see" (so "you may see error messages that are obscure" and/or "you may see red squiggles that end up under a random `...`").
```suggestion
messages that are obscure and/or red squiggles that end up under a random `...`. To avoid these, you might first 
```